dashboard/assets/packages/ui/src/types/config.ts

```diff
@@ -0,0 +1,18 @@
+// SPDX-FileCopyrightText: 2023 Raintank, Inc. dba Grafana Labs
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import { Section } from "@xk6-dashboard/view"
+
+export interface Tab {
```

Comment:
Could `@xk6-dashboard/view` export the Tab type?

Reply:
sure